e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -182,4 +182,22 @@ describe('Message', () => {
         // * Focus to remain in the RHS text box
         cy.get('#reply_textbox').should('be.focused');
     });
+
+    it('MM-T1797 Compact view: Show single image thumbnail', () => {
+        const filename = 'image-small-height.png';
+
+        // Make a post with some text and a single image
```

Comment:
Please change into our comment convention so that our non-technical staff can follow along what's going on with the script.
- `#` to indicate a test step
- `*` to indicate an assertion

```suggestion
        // # Make a post with some text and a single image
```